.github/workflows/ci.yml

```diff
@@ -0,0 +1,92 @@
+name: CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  linter:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Set up Python 3.10
```

Comment:
Why use 3.10 here if the minimum is 3.8? I think that mostly impacts e.g. the verison of a module like black which may have different behavior in other versions?

Reply:
I changed it